3-async-await/solution.js

```diff
@@ -1,35 +1,73 @@
-/*
-INSTRUCTIONS
-
-1. using async/await API consume products and prices methods
-2. don't use .then(), .catch() or .finally() here
-3. both, products and prices methods expect a positive integer id
-4. use Promise.all() and Promise.allSettled() to consume both methods in parallel
-5. to generate the id do the following: invoke Date.now(), and take the last two digits, this will be your id
-6. log the results with console.log(), the format is up to you, but it must include id, product and price
-
-Example:
-{
- id:100,
- product:'paper',
- price:1
-}
```

Comment:
We don't want to delete the instrucctions, so put them back and move your solution under them like other exercises.